sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -274,12 +274,53 @@ class MapboxMap :
     onMapLoadErrorListener: OnMapLoadErrorListener? = null,
   ) {
     checkNativeMap("loadStyle")
-    this.loadStyleUri(
-      styleExtension.styleUri,
-      transitionOptions,
-      { style -> onFinishLoadingStyleExtension(style, styleExtension, onStyleLoaded) },
-      onMapLoadErrorListener
+    initializeStyleLoad(
+      onStyleLoaded = { style ->
+        // TODO order of styleDataSourcesLoadedListener / styleDataSpritesLoadedListener is not defined
```

Comment:
imo worth creating public ticket for visibility and mention it here reducing comment size here in code

Reply:
Okay

Reply:
#1372